src/style.css

```
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.snake_block {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  width: 33.33%;
+}
+
+.snake__1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.snake__2 {
```

```suggestion
.snake_block:nth-child(2) {
```
Щоб не створювати купи лишніх класів-модифікаторв, краще використоувати псевдоклас nth-child